From 040b26068f493d07b1c2c3f168d5a6201d13eb3b Mon Sep 17 00:00:00 2001
From: Ondrej Jirman <megi@xff.cz>
Date: Fri, 24 Jul 2026 19:24:45 +0200
Subject: [PATCH] ARM: dts: sun8i-a83t: Add USB OTG DMA register region

Describe the USB-OTG internal DMA engine's registers so the musb glue
can use it, following the A64 pattern.

Signed-off-by: Ondrej Jirman <megi@xff.cz>
---
 arch/arm/boot/dts/allwinner/sun8i-a83t.dtsi | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/allwinner/sun8i-a83t.dtsi b/arch/arm/boot/dts/allwinner/sun8i-a83t.dtsi
index 1b29f27a93b9..b3f4412cffe2 100644
--- a/arch/arm/boot/dts/allwinner/sun8i-a83t.dtsi
+++ b/arch/arm/boot/dts/allwinner/sun8i-a83t.dtsi
@@ -641,7 +641,8 @@ msgbox: mailbox@1c17000 {
 		usb_otg: usb@1c19000 {
 			compatible = "allwinner,sun8i-a83t-musb",
 				     "allwinner,sun8i-a33-musb";
-			reg = <0x01c19000 0x0400>;
+			reg = <0x01c19000 0x0400>, <0x01c19500 0x0100>;
+			reg-names = "mc", "dma";
 			clocks = <&ccu CLK_BUS_OTG>;
 			resets = <&ccu RST_BUS_OTG>;
 			interrupts = <GIC_SPI 71 IRQ_TYPE_LEVEL_HIGH>;
-- 
2.43.0

